www/js/lib/zimArchive.js

```diff
@@ -20,8 +20,8 @@
  * along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
  */
 'use strict';
-define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
-    function(zimfile, zimDirEntry, util, utf8) {
+define(['zimfile', 'zimDirEntry', 'util', 'utf8', 'uiUtil'],
+    function(zimfile, zimDirEntry, util, utf8, uiUtil) {
```

Comment:
In terms of software architecture, we should not have a backend library like this one that depends on the UI.
The issue does not come from this PR. It was already there, it's just more visible in the code.
It's a technical debt we have for long: I was mentioning it in #43 (long time ago!)

(the same applies to zimArchiveLoader)

It should not be a blocker for this PR: I'll create a separate issue.